dev/release/update_change_log.sh

```diff
@@ -50,14 +50,14 @@ OUTPUT_PATH="${PROJECT}/CHANGELOG.md"
 pushd ${SOURCE_TOP_DIR}
 
 # reset content in changelog
-git co "${SINCE_TAG}" "${OUTPUT_PATH}"
+git checkout "${SINCE_TAG}" "${OUTPUT_PATH}"
 # remove license header so github-changelog-generator has a clean base to append
-sed -i '1,18d' "${OUTPUT_PATH}"
+sed -i.bak '1,18d' "${OUTPUT_PATH}"
```

Comment:
Likewise, apparently `sed` from mac is slightly different than linux 

https://stackoverflow.com/questions/5694228/sed-in-place-flag-that-works-both-on-mac-bsd-and-linux